packages/core/test/state.test.ts

```diff
@@ -716,7 +716,7 @@ describe('State', () => {
       expect(machine.initialState.can({ type: 'EV' })).toBe(true);
     });
 
-    it('should return false for a forbidden transition', () => {
+    it('should return true for a defined transition without a target', () => {
```

Comment:
We explicitly call this out in the docs as a "forbidden" transition and I think this should return `false`. When suggesting the simplified version of the check for `.can` I didn't realize that `.transitions` actually contain those forbidden transitions. In such a case the check should be slightly modified to account for this situation.

Reply:
Done